release-1.8/release_notes_draft.md

```diff
@@ -250,6 +271,18 @@ kind.
 #### kube-proxy ipvs mode
 * [alpha] Support ipvs mode for kube-proxy([#46580](https://github.com/kubernetes/kubernetes/pull/46580), [@haibinxie](https://github.com/haibinxie))
 
+### API Machinery
+
+* [alpha] The CustomResourceDefinition API can now optionally perform validation
```

Comment:
including links to docs like: https://github.com/nikhita/kubernetes.github.io/pull/1 ?

Reply:
And now I see that isn't open against the official repo yet.

Reply:
@deads2k It's already been merged. :)
This is the PR: https://github.com/kubernetes/kubernetes.github.io/pull/5290
And this would be the link to the docs: https://github.com/kubernetes/kubernetes.github.io/blob/release-1.8/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md#validation